src/main/scala/no/ndla/articleapi/model/api/UpdatedArticle.scala

```diff
@@ -27,17 +27,17 @@ case class UpdatedArticle(@(ApiModelProperty@field)(description = "The title of
                          )
 
 @ApiModel(description = "Information about the article")
-case class UpdatedArticleV2(@(ApiModelProperty@field)(description = "The title of the article") title: String,
-                            @(ApiModelProperty@field)(description = "The revision number for the article") revision: Int,
-                            @(ApiModelProperty@field)(description = "The content of the article") content: String,
+case class UpdatedArticleV2(@(ApiModelProperty@field)(description = "The revision number for the article") revision: Int,
+                            @(ApiModelProperty@field)(description = "The chosen language") language: String,
+                            @(ApiModelProperty@field)(description = "The title of the article") title: Option[String],
```

Comment:
Her gjeninnfører du `Option` som jeg føler har blitt fjernet mer eller mindre "overalt" som følge av språkendringen... Care to share why?

Reply:
I dette tilfellet for `UpdatedArticleV2` skal i utgangspunktet _alle_ feltene være optional (med noen unntak). Det brukes i PATCH endepunktet for å oppdatere en artikkel. Idéen her er at dersom man vil oppdatere bare tittelen, sender man bare inn tittel-feltet, ikke alle andre feltene. Jeg vet ikke hvorfor `title` og de noen av de andre feltene ikke var `Option`s, men det skal de altså være.

For `NewArticleV2` (som brukes for å opprette en ny artikkel) er fremdeles `title`, `content`, osv. påkrevd.

Forskjellen fra V1 verjsonene er at disse feltene har vært sekvenser av tittler, mens det nå er ett enkelt innslag.

I case classen som representerer artikkelen i sin helhet, som man får når man henter en artikkel med `GET`, er også `title`, `content`, osv. påkrevd

Reply:
Mer en god nok :)